.github/workflows/functional-test-noncloud.yaml

```diff
@@ -462,6 +504,43 @@ jobs:
           TEMP_CERT_DIR: ${{ steps.create-local-registry.outputs.temp-cert-dir }}
           SSL_CERT_FILE: ${{ steps.create-local-registry.outputs.temp-cert-dir }}/certs/${{ env.LOCAL_REGISTRY_SERVER }}/client.crt
 
+      - name: Verify Git server port-forward
```

Comment:
Why is step here instead of after the port-forward step?
Should this logic be included in the reusable action? 
Is this logic necessary? Are there circumstances where the port-forwarding would fail due to Radius code?
Note: this type of logic is not in the long running test workflow.